docs/make.jl

```diff
@@ -131,9 +131,9 @@ makedocs(;
             ExamplesPages...
         ],
         "Contributing" => [
-            "Overview" => "contributing/overview.md",
+            "Contribution guide" => "contributing/guide.md",
+            "Contribution guidelines"=> "contributing/guidelines.md",
             "Contributing to the documentation" => "contributing/new-documentation.md",
-            "Contributing to the dependencies" => "contributing/new-package.md",
```

Comment:
Why delete the whole page? IMO it was helpful, minimalist guide on creating new packages, which wasn't easy to find elsewhere online (it was a bit unstructured back then when I was looking into it). If we delete it, let's at least provide another link to fill the gap. Otherwise, our organization members might have trouble finding the info to quickly create a new package in the ecosystem. What do you think @albertpod

Reply:
I think Julia's "official" reference (https://julialang.org/contribute/developing_package/) is quite good as well

Reply:
Agree with @wouterwln